app/javascript/components/public_health/CustomExport.js

```diff
@@ -0,0 +1,508 @@
+                  checked={this.state.preset?.config?.data?.laboratories?.checked}
+                  expanded={this.state.preset?.config?.data?.laboratories?.expanded}
+                  onCheck={checked => this.handlePresetChange('config.data.laboratories.checked', checked)}
+                  onExpand={expanded => this.handlePresetChange('config.data.laboratories.expanded', expanded)}
+                  showNodeIcon={false}
+                  icons={rctIcons}
+                />
+              </Col>
+            </Row>
+            <Row className="mx-3 py-1 g-border-top">
+              <Col md={24} className="p-1">
+                <CheckboxTree
+                  id="rct-close-contacts-elements"
+                  nodes={this.props.options?.close_contacts?.nodes}
+                  checked={this.state.preset?.config?.data?.close_contacts?.checked}
+                  expanded={this.state.preset?.config?.data?.close_contacts?.expanded}
+                  onCheck={checked => this.handlePresetChange('config.data.close_contacts.checked', checked)}
+                  onExpand={expanded => this.handlePresetChange('config.data.close_contacts.expanded', expanded)}
+                  showNodeIcon={false}
+                  icons={rctIcons}
+                />
+              </Col>
+            </Row>
+            <Row className="mx-3 py-1 g-border-top">
+              <Col md={24} className="p-1">
+                <CheckboxTree
+                  id="rct-transfers-elements"
+                  nodes={this.props.options?.transfers?.nodes}
+                  checked={this.state.preset?.config?.data?.transfers?.checked}
+                  expanded={this.state.preset?.config?.data?.transfers?.expanded}
+                  onCheck={checked => this.handlePresetChange('config.data.transfers.checked', checked)}
+                  onExpand={expanded => this.handlePresetChange('config.data.transfers.expanded', expanded)}
+                  showNodeIcon={false}
+                  icons={rctIcons}
+                />
+              </Col>
+            </Row>
+            <Row className="mx-3 py-1 g-border-top">
+              <Col md={24} className="p-1">
+                <CheckboxTree
+                  id="rct-histories-elements"
+                  nodes={this.props.options?.histories?.nodes}
+                  checked={this.state.preset?.config?.data?.histories?.checked}
+                  expanded={this.state.preset?.config?.data?.histories?.expanded}
+                  onCheck={checked => this.handlePresetChange('config.data.histories.checked', checked)}
+                  onExpand={expanded => this.handlePresetChange('config.data.histories.expanded', expanded)}
+                  showNodeIcon={false}
+                  icons={rctIcons}
+                />
+              </Col>
+            </Row>
+          </div>
+          <hr className="m-0" />
+          <div className="p-2">
+            <h5 className="mx-3 my-2">Custom Export Format Name</h5>
```

Comment:
Small thing - this section title is all capitalized and the other sections aren't. I don't have a strong preference what it should be overall, but I'd like them to be consistent.

Reply:
I too would like consistency, this text/language is just going strictly off of what Kelly provided in the mockup

Reply:
Please make this `Custom export format name`, to match the other sections.

Reply:
I am not a fan of allowing arbitrary filenames like this, no matter how much sanitization or safe handling we attempt to do (someone could make a change 5 years down the road that could break whatever we are ensuring with this right now). I'd rather just never even do this in the first place. Either allow someone to pick and choose some things (like timestamp, workflow, their username, etc.), or remove this functionality completely.

Reply:
Just to throw this out there (for when we merge in Kyle's active storage changes - definitely not in this PR), we should add some visual feedback if someone selected CSV instead of Excel, as that would result in multiple files versus one excel document.

Reply:
sounds good, I will remove the ability to customize the filename for now

Reply:
https://github.com/SaraAlert/SaraAlert/pull/562/commits/18f2cd1c1295b23fca3eb469bf542092217c6acb#diff-c34521fcd0fdc2034d456886aae8fd35299fe908ab88d8e49c60d0b4d39ab26bR822

Reply:
https://github.com/SaraAlert/SaraAlert/pull/562/commits/18f2cd1c1295b23fca3eb469bf542092217c6acb#diff-14793b3bc71143b45317a3d08474d370db9ffdfb9643e754341eeda1ab5f096bR387

Reply:
this 'Custom export format name' will now just be the preset name instead of being the filename as well